samples/cellular/modem_shell/src/link/link_shell.c

```diff
@@ -1997,6 +2043,85 @@ static int link_shell_redmob(const struct shell *shell, size_t argc, char **argv
 	return 0;
 }
 
+static int link_shell_uiccpowersave(const struct shell *shell, size_t argc, char **argv)
+{
+	int ret;
+	enum link_shell_operation operation = LINK_OPERATION_NONE;
+	enum lte_lc_uiccpowersave_mode uiccpowersave_mode = LINK_UICCPOWERSAVE_NONE;
+	char snum[10];
+
+	optreset = 1;
+	optind = 1;
+	int opt;
+
+	while ((opt = getopt_long(argc, argv, short_options, long_options, NULL)) != -1) {
+		switch (opt) {
+		case 'r':
+			if (operation != LINK_OPERATION_NONE) {
```

Comment:
This branch should be unnecessary. Just one of the operations returned by link_shell_getopt_operation() is possible so you cannot really get here.